scripts/communication.js

```diff
@@ -3,26 +3,54 @@
 //   If somebody in chat write appropriate query,
 //   mona will answer him a random response for this query.
 
-const greetings = ['Hello', 'Good morning', 'G\'day', '\^Hi\$', 
-  'Доброго утра', 'Доброе утро', 'Привет'];
+const incomingGreetings = [
+  'Hello',
+  'Good morning',
+  'G\'day',
+  '\^Hi\$',
+  'Morning',
+  'Hey',
+  'Wazzzup',
```

Comment:
I just hate "wazzzup" word since I remember a video with it https://www.youtube.com/watch?v=Kh_AMVgkAjk but that's my personal thing :).

Reply:
@dzhlobo thoughts about adding 
```
"Absolutely."
"I shall not fear."
"As the Goddess wills."
"I walk in the shadow."
"My wait is over."
```
huntress' quotes from [there](https://worldofwarcraft.fandom.com/et/wiki/Quotes_of_Warcraft_III/Night_Elf_Sentinels) 🙂

Reply:
@DDKatch that's greeting stuff, `"Absolutely."` and others don't fit here

Reply:
I see what you meant 😄

Reply:
@DDKatch, they don't sound as a reply to "hi"?

Reply:
@dzhlobo actually it should be instead of `yes` 😉